packages/expo-video/src/VideoView.tsx

```diff
@@ -7,9 +7,10 @@ import type { VideoViewProps } from './VideoView.types';
 
 /**
  * Returns whether the current device supports Picture in Picture (PiP) mode.
+ *
+ * > **Note:** All major web browsers, except Firefox, support Picture in Picture (PiP) mode.
```

Comment:
```suggestion
 * > **Note:** All major web browsers support Picture in Picture (PiP) mode except Firefox.
```

Let's add the exception at the end so the sentence does not require any pauses.